messaging-activemq/src/test/java/org/wildfly/extension/messaging/activemq/BridgeAddTestCase.java

```diff
@@ -0,0 +1,210 @@
+        model.get("call-timeout").set(CALL_TIMEOUT);
+        model.get("static-connectors").add(CONNECTOR_NAME);
+
+        BridgeConfiguration bridgeConfiguration = BridgeAdd.createBridgeConfiguration(context, BRIDGE_NAME, model);
+
+        Assert.assertEquals(BRIDGE_NAME, bridgeConfiguration.getName());
+        Assert.assertEquals(SOURCE_QUEUE_NAME, bridgeConfiguration.getQueueName());
+        Assert.assertEquals(CALL_TIMEOUT, bridgeConfiguration.getCallTimeout());
+        Assert.assertEquals(CONNECTOR_NAME, bridgeConfiguration.getStaticConnectors().get(0));
+    }
+
+    /**
+     * Verifies that {@link BridgeAdd#createBridgeConfiguration(OperationContext, String, ModelNode)} creates
+     * a BridgeConfiguration instance with callTimeout attribute correctly set via the system property.
+     */
+    @Test
+    public void testCreateBridgeConfiguration_callTimeoutSetViaSystemProperty() throws OperationFailedException {
+        try {
+            System.setProperty(BridgeAdd.CALL_TIMEOUT_PROPERTY, String.valueOf(CALL_TIMEOUT));
+
+            ModelNode model = new ModelNode();
+            model.get("queue-name").set(SOURCE_QUEUE_NAME);
+            model.get("static-connectors").add(CONNECTOR_NAME);
+
+            BridgeConfiguration bridgeConfiguration = BridgeAdd.createBridgeConfiguration(context, BRIDGE_NAME, model);
+
+            Assert.assertEquals(BRIDGE_NAME, bridgeConfiguration.getName());
+            Assert.assertEquals(SOURCE_QUEUE_NAME, bridgeConfiguration.getQueueName());
+            Assert.assertEquals(CALL_TIMEOUT, bridgeConfiguration.getCallTimeout());
+            Assert.assertEquals(CONNECTOR_NAME, bridgeConfiguration.getStaticConnectors().get(0));
+        } finally {
+            System.clearProperty(BridgeAdd.CALL_TIMEOUT_PROPERTY);
+        }
+    }
+
+
+    /**
+     * Verifies that {@link BridgeAdd#createBridgeConfiguration(OperationContext, String, ModelNode)} creates
+     * a BridgeConfiguration instance with callTimeout attribute set to its' default value, when it's not modified via the management model
+     * attribute or the system property.
+     */
+    @Test
+    public void testCreateBridgeConfiguration_callTimeoutUnset() throws OperationFailedException {
+        ModelNode model = new ModelNode();
+        model.get("queue-name").set(SOURCE_QUEUE_NAME);
+        model.get("static-connectors").add(CONNECTOR_NAME);
+
+        BridgeConfiguration bridgeConfiguration = BridgeAdd.createBridgeConfiguration(context, BRIDGE_NAME, model);
+
+        Assert.assertEquals(BRIDGE_NAME, bridgeConfiguration.getName());
+        Assert.assertEquals(SOURCE_QUEUE_NAME, bridgeConfiguration.getQueueName());
+        Assert.assertEquals(ActiveMQClient.DEFAULT_CALL_TIMEOUT, bridgeConfiguration.getCallTimeout());
+        Assert.assertEquals(CONNECTOR_NAME, bridgeConfiguration.getStaticConnectors().get(0));
+    }
+
+    @Test
+    public void testPerformRuntime() throws Exception {
```

Comment:
Doesn't have much value and is covered by the integration testing

Reply:
File removed.